client/src/components/Search/Search.jsx

```diff
@@ -94,12 +158,37 @@ export default function Search() {
             timeFormat="HH:00"
             dateFormat="MMMM d, yyyy h:mm aa"
             showTimeSelect
-          />
+          /> */}
+          {/* select date */}
+          <DatePicker selected={selectedDate} 
+          placeholderText="Date?" 
+          onChange={date => setSelectedDate(date)} />
+          {/* select start time */}
+          <DatePicker
+          selected={selectedStartTime}
+          placeholderText="Start time?" 
+          onChange={startTime => setSelectedStartTime(startTime)}
+          showTimeSelect
+          showTimeSelectOnly
+          timeIntervals={15}
+          timeCaption="Time"
+          dateFormat="h:mm aa"
+            />
```

Comment:
this should be in a separate div with className="input"

Reply:
Thx! I did it!